### ISSUE TITLE
[New Component] Observability 

### ISSUE THREAD

**Issue by tarun-aiplanet** [aiplanethub/beyondllm#31]
Track the latency, cost other metrics. 

- [x] Phoenix: #47
- [ ] Langfuse
- [ ] MLFlow

**Comment by dapphari007** [comment]
Shall I try to do my contribution to this task...

**Comment by jaintarunAI** [comment]
Sure. Go ahead @dapphari007 

Can you assign which tool you need to integrate?

**Comment by dapphari007** [comment]
Yeah broo... I'm sorry I didn't receive any notifications of this. I'm just a beginner I will try what the things to do. if u say I will do with the help of references from internet for your project... I'm really sorry bro... I will definitely contact you 
Here is my mail id : dappinmonster007@gmail.com
I will be there....

**Comment by dapphari007** [comment]
Bro !!!

**Comment by jaintarunAI** [comment]
> Bro !!!

The tool name is already mentioned in the list. Please find the similar PR for the reference: #47 

This is Open Source, If you are beginner, you need to get hold of building RAG pipeline first.
